assets/src/edit-story/components/canvas/karma/multiSelectionMovable.karma.js

```diff
@@ -0,0 +1,171 @@
+          font: TEXT_ELEMENT_DEFAULT_FONT,
+          content: 'Text A',
+          x: 10,
+          y: 10,
+          width: 50,
+          height: 50,
+        })
+      );
+
+      element2 = await fixture.act(() =>
+        insertElement('text', {
+          font: TEXT_ELEMENT_DEFAULT_FONT,
+          content: 'Text B',
+          x: 100,
+          y: 100,
+          width: 50,
+          height: 50,
+        })
+      );
+
+      element3 = await fixture.act(() =>
+        insertElement('text', {
+          font: TEXT_ELEMENT_DEFAULT_FONT,
+          content: 'Text B',
+          x: element1.x + element1.width + 1,
+          y: element1.y + element1.height + 1,
+          width: 20,
+          height: 20,
+        })
+      );
+      frame3 = fixture.querySelector(
+        `[data-element-id="${element3.id}"] [data-testid="textFrame"]`
+      );
+
+      frame1 = fixture.querySelector(
+        `[data-element-id="${element1.id}"] [data-testid="textFrame"]`
+      );
+
+      frame2 = fixture.querySelector(
+        `[data-element-id="${element2.id}"] [data-testid="textFrame"]`
+      );
+    });
+
+    it('should render initial content', () => {
+      expect(frame1.textContent).toEqual('Text A');
+      expect(frame2.textContent).toEqual('Text B');
+    });
+
+    it('should multi-select elements on Shift + click', async () => {
+      await fixture.events.click(frame1);
+      await fixture.events.keyboard.down('Shift');
+      await fixture.events.click(frame2);
+      await fixture.events.keyboard.up('Shift');
+
+      const storyContext = await fixture.renderHook(() => useStory());
+      expect(storyContext.state.selectedElementIds).toEqual([
+        element1.id,
+        element2.id,
+      ]);
+    });
+
+    describe('click interaction', () => {
+      let safezone;
+      beforeEach(async () => {
+        await fixture.events.click(frame1);
+        await fixture.events.keyboard.down('Shift');
+        await fixture.events.click(frame2);
+        await fixture.events.keyboard.up('Shift');
+
+        safezone = fixture.querySelector('[data-testid="safezone"]');
+      });
+
+      it('should select one element when clicking in multi-selection', async () => {
+        const { x, y } = safezone.getBoundingClientRect();
+        await fixture.events.mouse.click(
+          x + element1.x + 1,
+          y + element1.y + 1
+        );
+        const storyContext = await fixture.renderHook(() => useStory());
+        expect(storyContext.state.selectedElementIds).toEqual([element1.id]);
+      });
+
+      it('should select bg element when clicking out of the multi-selection', async () => {
+        const { x, y } = safezone.getBoundingClientRect();
+        await fixture.events.mouse.click(x + 200, y + 200);
+        const storyContext = await fixture.renderHook(() => useStory());
+
+        // Let's assure we have the background element (the first element) in the selection now.
+        const background = safezone.firstChild;
```

Comment:
Hmm. This is the same issue and even a bit more pronounced. Do we really need it? Would it be easier to just get the background element from the storyContext?